<launch>
    <arg name="visualization" default="true"/>
    <arg name="location" default="sargans"/>
    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>

    <node pkg="grid_map_geo" type="test_tif_loader" name="test_tif_loader" output="screen">
        <param name="tif_path" value="$(find terrain_models)/models/dischma_valley.tif"/>
        <param name="color_path" value="$(find terrain_models)/models/dischma_valley_color.tif"/>
    </node>

    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find grid_map_geo)/launch/config.rviz" />
    </group>
</launch>
